### PR TITLE
Add Fabric8 demo data

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

File: demo-data/src/main/java/org/jboss/pnc/demo/data/DatabaseDataInitializer.java [hunk 1]
```diff
@@ -131,10 +132,15 @@ public void initiliazeData() {
                     .name("JBoss JavaEE Servlet Spec API").description("JavaEE Servlet Spec API")
                     .projectUrl("https://github.com/jboss/jboss-servlet-api_spec")
                     .issueTrackerUrl("https://issues.jboss.org/browse/JBEE").build();
+            Project project4 = Project.Builder.newBuilder()
```
Comment (slaskawi) [inline demo-data/src/main/java/org/jboss/pnc/demo/data/DatabaseDataInitializer.java]: I think we may try to remove some of those projects and start replacing it with proper ones (like Fabric8)

File: demo-data/src/main/java/org/jboss/pnc/demo/data/DatabaseDataInitializer.java [hunk 2]
```diff
@@ -177,6 +193,9 @@ public void initiliazeData() {
                     .buildConfiguration(buildConfiguration3)
                     .productVersion(productVersion).build();
 
+            BuildConfigurationSet buildConfigurationSet2 = BuildConfigurationSet.Builder.newBuilder().name("Build Config Set 2")
```
Comment (slaskawi) [inline demo-data/src/main/java/org/jboss/pnc/demo/data/DatabaseDataInitializer.java]: Maybe there is a better name, like Fabric8....

**Comment by slaskawi** [comment]
Could you please also add a JIRA reference to the commit msg?

**Comment by slaskawi** [comment]
retest this please

**Comment by rnc** [comment]
retest this please

**Comment by rnc** [comment]
retest this please

**Pull request merged**